tests/test_project_listing.py

```diff
@@ -0,0 +1,104 @@
+# IMPORTS ################################################################################ IMPORTS #
+
+# Standard library
+import http
+import datetime
+import json
+
+# Own
+from dds_web import db
+from dds_web.database import models
+import tests
+
+
+# CONFIG #################################################################################### CONFIG #
+
+
+proj_data = {"pi": "piName", "title": "Test proj", "description": "A longer project description"}
+proj_query = {"project": "public_project_id"}
+proj_query_restricted = {"project": "restricted_project_id"}
+# TESTS #################################################################################### TESTS #
```

Comment:
I'd put a newline on top of the # TESTS #### ... part. That's personal preference though.

Reply:
Not on purpose. Will fix it together with the other stuff...